internal/super/flix.go

```diff
@@ -47,6 +47,13 @@ type flixFlags struct {
 	Include     []string `default:"" flag:"include" info:"Fields to include in the output"`
 	Exclude     []string `default:"" flag:"exclude" info:"Fields to exclude from the output (events)"`
 	GasLimit    uint64   `default:"1000" flag:"gas-limit" info:"transaction gas limit"`
+	Bindings    string   `default:"" flag:"bindings" info:"generate binding code for the given language, fcl-js' is supported language"`
+	Save        string   `default:"" flag:"output" info:"output file for generated binding code"`
```

Comment:
there is a global save flag on all flow commands, can we not use that one here?

Reply:
There is also a global --output flag and this would clash with that.

Reply:
good call, I'll update to use the flag `--save` also pushed a change to reorganize flix verbs. I'll update PR description.